pyuvdata/tests/test_ms.py

```diff
@@ -39,12 +40,33 @@ def test_readNRAO():
     """Test reading in a CASA tutorial ms file."""
     UV = UVData()
     testfile = os.path.join(DATA_PATH, 'day2_TDEM0003_10s_norx_1src_1spw.ms')
-    expected_extra_keywords = ['data_column', 'antenna_positions']
+    expected_extra_keywords = ['DATA_COL']
```

Comment:
It was a bug in the test code. A few lines later the lists are being called with a `.sort()` method, which returns `None` for some reason, so the test was passing. The right thing is to call `sorted` on the list. I fixed that and then had to fix this because it was incorrect.